lib/configure.js

```diff
@@ -137,6 +139,18 @@ function configure (gyp, argv, callback) {
     // disable -T "thin" static archives by default
     variables.standalone_static_library = gyp.opts.thin ? 0 : 1
 
+    if (win && !(gyp.opts.msvs_version && gyp.opts.msvs_version !== '2017')) {
+      const vsSetup = findVS2017()
+      if (vsSetup) {
+        gyp.opts.msvs_version = '2015'
```

Comment:
We are forcing this to make GYP happy, and to avoid pulling https://chromium-review.googlesource.com/#/c/433540/ (which did not land yet) as a floating patch, since floating patches seem to be avoided here.